chemprop/args.py

```diff
@@ -128,6 +133,10 @@ def process_args(self) -> None:
         if self.features_generator is not None and 'rdkit_2d_normalized' in self.features_generator and self.features_scaling:
             raise ValueError('When using rdkit_2d_normalized features, --no_features_scaling must be specified.')
 
+        # Validate atom descriptors
+        if self.atom_descriptors != 'None' and self.atom_descriptors_path is None:
```

Comment:
Don't forget to change `self.atom_descriptors != 'None'` to `self.atom_descriptors is not None` if you decide to make the change I suggested earlier.